docs/source/annotated_examples.rst

```diff
@@ -17,7 +17,7 @@ examples as starting points for their own applications.
 -----------------------
 An example showing Chaco's BarPlot class.
 
-source: `bar_plot_stacked.py <https://github.com/enthought/chaco/tree/main/examples/demo/basic/bar_plot_stacked.py>`_
+source: :github-demo:`bar_plot_stacked.py <chaco/examples/demo/basic/bar_plot_stacked.py>`.
```

Comment:
I haven't met the `github-demo` role before. Where is this defined? (Is there documentation somewhere?)

Reply:
it's a custom role that we define - https://github.com/enthought/chaco/blob/737bcc73c8222cf3a236cc6feee5103b41f88f10/docs/source/conf.py#L227-L231